app/controllers/articles_controller.rb

```diff
@@ -29,6 +30,7 @@ def update
 
   def show
     @article = Article.find(params[:id])
```

Comment:
cancan should fill @article out for you if you have load_and_authorize_resource at the top of this file